packages/cadl-ranch-specs/http/azure/core/mockapi.ts

```diff
@@ -2,7 +2,7 @@ import { passOnSuccess, mockapi, json, ValidationError } from "@azure-tools/cadl
 import { ScenarioMockApi } from "@azure-tools/cadl-ranch-api";
 
 export const Scenarios: Record<string, ScenarioMockApi> = {};
-
+const validUser = { id: 1, name: "Madge", ETag: "11bdc430-65e8-45ad-81d9-8ffa60d55b59" };
```

Comment:
All "etag" should be in lower case?
https://github.com/Azure/typespec-azure/blob/main/packages/typespec-azure-core/lib/models.tsp#L126-L134